cms/src/test/resources/test-data/api-specifications/SchemaHelperTest/schemaObjectCompleteTopLevel-simpleFields.json

```diff
@@ -0,0 +1,40 @@
+{
+    "openapi": "3.0.0",
+    "paths": {
+        "/test": {
+            "post": {
+                "requestBody": {
+                    "content": {
+                        "application/json": {
+
+                            "schema": {
+                                "type": "object",
+                                "title": "Test Schema Title",
+                                "deprecated": true,
+                                "description": "Test Schema Object description in `Markdown`.",
+                                "multipleOf": 1.2,
+                                "minimum": -2.3,
+                                "exclusiveMinimum": true,
+                                "maximum": 2.2,
+                                "exclusiveMaximum": false,
+                                "minLength": 11,
+                                "maxLength": 22,
+                                "pattern": "^(?:[0-9]{4}-[0-9]{2}-[0-9]{2})?(?:[ T][0-9]{2}:[0-9]{2}:[0-9]{2})?(?:[.,][0-9]{3})?",
```

Comment:
Can a comment indicate what data type this is looking for?  I assume it's a date that can be empty.

Reply:
The short and cheeky answer is: I don't know and I don't care :grin: 

The reason is that, in context of this test (and in the rest of our implementation in the CMS), we don't use this value as a regex to do any matching; instead, all we do with it is just to display it on the website as is.

[OAS specification](https://spec.openapis.org/oas/v3.0.3#schema-object) requires that the string is a regex - and so I used a rather elaborate expression here but only to see that it is included in the HTML rendered by `SchemaHelper` class.